app/client/tests/unit/worker.spec.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs';
+
+const vm = require('vm');
```

Comment:
could this one be imported the same way as `fs`? then the `no-vars-require` rule could be left on: https://github.com/bacpop/beebop/pull/7/files#diff-b8d2936a034f3ef827722f0cd18b81d5522b816bfe559ce499e8841f2142aeafR81

Reply:
I tried, but I would then get an error : `Argument of type 'Buffer' is not assignable to parameter of type 'string'` for `data` in [line 6](https://github.com/bacpop/beebop/blob/d588b3743463882384a50498956eee262798ccb1/app/client/tests/unit/worker.spec.ts#L6).
It is also working the other way round (requiring fs, importing vm), but not when both are imported...

Reply:
Woah, that's odd. Let me take a look and see if I can figure out why

Reply:
Ok, this is because it's a typescript error, but if one of the packages is imported with `require` it can't perform type checking.

The fix is:

```
import fs from 'fs';
import vm from 'vm';

const data = fs.readFileSync('./public/worker.js', { encoding: 'utf8' });
```